lib/chat_api/conversations.ex

```diff
@@ -37,11 +39,13 @@ defmodule ChatApi.Conversations do
     |> Repo.all()
   end
 
+  @spec list_conversations_by_account(integer()) :: [Converation.t()]
```

Comment:
typo (`Converation` -> `Conversation`)

also `account_id` is a UUID, not an integer -- sorry again for the confusion!